Cargo.toml

```diff
@@ -2,5 +2,6 @@
 members = ["bpfd", "bpfd-api", "bpfd-common", "bpfctl", "xtask", "tests/integration-test", "tests/integration-test-macros"]
 
 [patch.crates-io]
-aya = { git = "https://github.com/redhat-et/aya", branch="main" }
+# aya = { git = "https://github.com/redhat-et/aya", branch="main" }
+aya = { path = "/home/afredette/projects/rh-aya/aya" }
```

Comment:
This will need to be updated or reverted depending on where the needed commits end up first.